.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+# Description:
+
+______________
+
+# Tasks:
+
+- [ ] Describe the problem / feature
+- [ ] Write tests
+- [ ] Write code to solve the problem
+- [ ] Get code review from coworkers / friends
+- [ ] [Squash commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)
+
+I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
No I can not, I'm not maintainer and it's not my place to know such situations, I was asked by one of the maintainers to add it - and I also believe it fits here good.

Please read https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md the line added here only ensures it's known to everybody before something that would invalidate their contribution.